package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@ironcorelabs/recrypt-node-binding",
-    "version": "0.4.0",
+    "version": "0.4.1",
```

Comment:
Bump the version in the `Cargo.toml` file as well. Also, you might want to fill out a changelog entry for this too.